x-pack/plugins/ml/public/application/data_frame_analytics/pages/analytics_creation/components/create_step/create_step.tsx

```diff
@@ -82,6 +86,7 @@ export const CreateStep: FC<Props> = ({ actions, state, step }) => {
       )}
       <EuiSpacer size="s" />
       <Messages messages={requestMessages} />
+      {showProgress && <ProgressStats jobId={jobId} />}
```

Comment:
Wondering how a 'Progress' sub-heading might look above this section?

![image](https://user-images.githubusercontent.com/7405507/85422477-8ffce600-b56d-11ea-8e17-3f5e4a5bc2a1.png)


In the Transforms wizard, we have the label above the progress bar form control, but perhaps a sub-heading might look better here? Or maybe we don't need one at all!?

Reply:
Added subheading in 9bc31924120d1749680a4d2a25e64a617408774b